tutorials/plugins/gdnative/gdnative-cpp-example.rst

```diff
@@ -388,6 +485,348 @@ We're finally ready to run the project:
 
 .. image:: img/gdnative_cpp_animated.gif
 
+Adding properties
+-----------------
```

Comment:
Nitpick, but we usually put be an empty newline after a heading :)

Reply:
I've found sphynx to just spit the dummy without the empty line, so yes, add :)